sdk/js/src/service/gateways.js

```diff
@@ -184,6 +186,33 @@ class Gateways {
     // Combine all stream sources to one subscription generator.
     return combineStreams(streams)
   }
+
+  // Gateway Configuration Server
```

Comment:
```suggestion
  // Gateway Configuration Server.
```

https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/DEVELOPMENT.md#code-comments

I recommend you to (re)familiarize yourself with our guidelines. Happy for suggestions for what else to include here as well.